### PR TITLE
(FIXUP) Make system config path absolute instead of relative on POSIX hosts

### PR DESCRIPTION
**Pull request by scotje**
Pretty much what it says on the tin.

### CODE REVIEW

File: spec/unit/pdk/util_spec.rb [hunk 1]
```diff
@@ -349,7 +349,7 @@
       end
 
       it 'returns a path inside the system opt directory' do
-        is_expected.to eq(File.join('opt', 'puppetlabs', 'pdk', 'config'))
+        is_expected.to match(%r{\A/opt/})
```
Comment (scotje) [inline spec/unit/pdk/util_spec.rb]: We can make this more specific if we want, but this now test now better matches it's description.

**Comment by rodjek** [comment]
We should look at getting some acceptance tests going for this as well

**Comment by glennsarti** [comment]
@rodjek Did you want the acceptance tests in this PR?

**Comment by glennsarti** [comment]
I've added some tests in #859 9https://github.com/puppetlabs/pdk/pull/859/commits/a3b68dbcb8928d0bef3972f30bff4a30cc6222dc) Which should cover this.

**Pull request merged**